internal/agent/utils.go

```diff
@@ -0,0 +1,35 @@
+package agent
+
+import "encoding/base64"
+
+// ObfuscationResult represents the result of an obfuscation request
+type ObfuscationResult struct {
+	ObfuscatedValue string
```

Comment:
We try and stick to camel case for JSON output values.  You can achieve that here by adding a struct tag:

```go
type ObfuscationResult struct {
	ObfuscatedValue string `json:"obfuscatedValue"`
}
```